app/src/main/res/drawable/rounded_game_card.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <solid android:color="@color/card" />
+    <corners android:radius="16dp" />
```

Comment:
why attributes have hardcode please change to resource value

Reply:
Done